src/components/vm/vmCloneDialog.jsx

```diff
@@ -95,7 +95,7 @@ export const CloneDialog = ({ name, connectionName }) => {
                     <TextInput id='vm-name'
                                validated={validationFailed.name ? "error" : "default"}
                                value={newVmName}
-                               onChange={setNewVmName} />
+                               onChange={(_, value) => setNewVmName(value)} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230629-150027-4152f662-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)